generatedTypes/components/avatar/index.d.ts

```diff
@@ -63,6 +77,21 @@ export declare type AvatarProps = Pick<AccessibilityProps, 'accessibilityLabel'>
      * fails (equiv. to Image.onError()).
      */
     onImageLoadError?: ImagePropsBase['onError'];
+    /**
+     * The name of the avatar user.
+     * If no label is provided, the initials will be generated from the name.
+     * autoColorsConfig will use the name to create the background color of the Avatar.
+     */
+    name?: string;
+    /**
+     * Hash the name (or label) to get a color, so each name will have a specific color.
+     * Default is false.
+     */
+    useAutoColorsConfig?: boolean;
```

Comment:
`useAutoColorsConfig` sounds like the user must pass it. I think `useAutoColors` is better
```suggestion
    useAutoColors?: boolean;
```

Reply:
Done